wifiphisher/common/recon.py

```diff
@@ -120,7 +64,7 @@ def _process_packets(self, packet):
                 # to the list
                 # note \00 used for when ap is hidden and shows only the length
                 # of the name. see issue #506
-                if not packet.info or "\00" in packet.info:
+                if not packet.info or b"\00" in packet.info:
```

Comment:
Why did you do this change?

b"\00" and "\00" are different values.

Reply:
```python2
>>> a = "\x00"
>>> a
'\x00'
>>> b"\x00" in a
True
>>> "\x00" in a
True
```

Reply:
Seems right.